core/gui/src/app/common/service/notification/notification.service.ts

```diff
@@ -8,7 +9,18 @@ import { NzMessageDataOptions, NzMessageService } from "ng-zorro-antd/message";
   providedIn: "root",
 })
 export class NotificationService {
-  constructor(private message: NzMessageService) {}
+  constructor(
+    private message: NzMessageService,
+    private notification: NzNotificationService
+  ) {}
+
+  blank(title: string, content: string, options: NzMessageDataOptions = {}): void {
```

Comment:
I think there is no need to add this `blank`. You can remove it. Can you reuse `loading` to indicate the report generation>